src/main/java/lotto/domain/LottoStatistics.java

```diff
@@ -0,0 +1,67 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LottoStatistics {
+    public static final long THREE_MATCHING_PRIZE = 5000L;
+    public static final long FOUR_MATCHING_PRIZE = 50000L;
+    public static final long FIVE_MATCHING_PRIZE = 1500000L;
+    public static final long SIX_MATCHING_PRIZE = 2000000000L;
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
    public static final long SIX_MATCHING_PRIZE = 2_000_000_000L;
```